diff --git a/node_modules/react-native-reanimated/lib/reanimated2/core.js b/node_modules/react-native-reanimated/lib/reanimated2/core.js
index 706a560..117505d 100644
--- a/node_modules/react-native-reanimated/lib/reanimated2/core.js
+++ b/node_modules/react-native-reanimated/lib/reanimated2/core.js
@@ -1,7 +1,7 @@
 /* global _WORKLET _getCurrentTime _frameTimestamp _eventTimestamp, _setGlobalConsole */
 import NativeReanimatedModule from './NativeReanimated';
 import { Platform } from 'react-native';
-import { nativeShouldBeMock, shouldBeUseWeb } from './PlatformChecker';
+import { nativeShouldBeMock, shouldBeUseWeb, isWeb } from './PlatformChecker';
 global.__reanimatedWorkletInit = function (worklet) {
     worklet.__worklet = true;
 };
@@ -263,8 +263,8 @@ if (!NativeReanimatedModule.useOnlyV1) {
     NativeReanimatedModule.installCoreFunctions(NativeReanimatedModule.native
         ? workletValueSetter
         : workletValueSetterJS);
-    const capturableConsole = console;
-    isConfigured() &&
+    if (!isWeb() && isConfigured()) {
+        const capturableConsole = console;
         runOnUI(() => {
             'worklet';
             const console = {
@@ -279,6 +279,7 @@ if (!NativeReanimatedModule.useOnlyV1) {
                 now: global._chronoNow,
             };
         })();
+    }
 }
 let featuresConfig = {
     enableLayoutAnimations: false,
